src/cli/format.js

```diff
@@ -313,17 +312,17 @@ async function formatFiles(context) {
     }
   }
 
-  for await (const pathOrError of expandPatterns(context)) {
-    if (typeof pathOrError === "object") {
-      context.logger.error(pathOrError.error);
+  for await (const { error, fileName, ignoreUnknown } of expandPatterns(
```

Comment:
Why rename `filename` to `fileName`? Filename is one word.

Reply:
Because `expandPatterns` yields `fileName`. But I don't have any opinion on this naming.

Reply:
`filename` is fine when use alone, but when use together with `filePath` (I don't think `filepath` it's a word) and `fileUrl` , it's a little bit odd. So I use `fileName` sometime.

I'm not a native English speaker, not sure what's the best.

Reply:
And I use lots of `fileUrlOrPath` in recent PRs...